src/gyazo.rb

```diff
@@ -138,3 +157,5 @@
     File.open(idfile,"w").print(newid)
   end
 }
+
+File.delete(tmpfile)
```

Comment:
add newline at the end
```suggestion
File.delete(tmpfile)

```